api/include/opentelemetry/baggage/baggage.h

```diff
@@ -0,0 +1,377 @@
+        metadata = TrimString(list_member.substr(value_end_pos + 1));
+      }
+
+      int err    = 0;
+      auto key   = Decode(TrimString(list_member.substr(0, key_end_pos)), err);
+      auto value = Decode(TrimString(list_member.substr(key_end_pos + 1, value_end_len)), err);
+      if (IsValidKey(key) && IsValidValue(value) && err == 0)
+      {
+        Entry entry(key, value, metadata);
+        if (end - begin + 1 < kMaxKeyValueSize)
+        {
+          (baggage->entries_.get())[baggage->num_entries_] = entry;
+          baggage->num_entries_++;
+        }
+      }
+
+      begin = end + 2;
+    }
+
+    return baggage;
+  }
+
+  // Creates string from baggage object.
+  std::string ToHeader()
+  {
+    std::string header_s;
+    for (size_t count = 0; count < num_entries_; count++)
+    {
+      if (count != 0)
+      {
+        header_s.append(",");
+      }
+      auto entry = (entries_.get())[count];
+      header_s.append(Encode(entry.GetKey()));
+      header_s.push_back(kKeyValueSeparator);
+      header_s.append(Encode(entry.GetValue()));
+      if (entry.GetMetadata().size())
+      {
+        header_s.push_back(kValueMetadataSeparator);
+        header_s.append(std::string{entry.GetMetadata()});
+      }
+    }
+    return header_s;
+  }
+
+  // Access the value for a name/value pair set by a prior event. Returns a value associated with
+  // the given name or null if the given name is not present.
+  std::string Get(nostd::string_view key) const noexcept
+  {
+    if (!IsValidKey(key))
+    {
+      return std::string();
+    }
+    for (size_t i = 0; i < num_entries_; i++)
+    {
+      auto entry = (entries_.get())[i];
+      if (key == entry.GetKey())
+      {
+        return std::string(entry.GetValue());
+      }
+    }
+    return std::string();
+  }
+
+  // Record the value for a name/value pair. Returns a new Baggage that contains the new value.
+  nostd::shared_ptr<Baggage> Set(nostd::string_view key,
+                                 nostd::string_view value,
+                                 nostd::string_view metadata = "")
+  {
+    nostd::shared_ptr<Baggage> baggage{new Baggage()};
+    if (!IsValidKey(key) || !IsValidValue(value))
+    {
+      return baggage;
+    }
+
+    Entry new_e(key, value, metadata);
+    (baggage->entries_.get())[baggage->num_entries_++] = new_e;
+    for (size_t i = 0; i < num_entries_; i++)
+    {
+      // Each name in the baggage must be associated with only one value.
+      // We do not add old value for the key.
+      if (key == (entries_.get())[i].GetKey())
+      {
+        continue;
+      }
+
+      Entry e((entries_.get())[i]);
+      (baggage->entries_.get())[baggage->num_entries_++] = e;
+    }
+
+    return baggage;
+  }
+
+  // Delete a name/value pair. Returns a new Baggage which does not contain the selected key.
+  nostd::shared_ptr<Baggage> Remove(nostd::string_view key)
+  {
+    nostd::shared_ptr<Baggage> baggage(new Baggage{});
```

Comment:
At line 95, parentheses () are used and here uniform initialiser {} is used. I would say lets prefer single format, though it won't make any difference in functionality.